shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/pool/metadata/DataSourceJdbcUrlMetaData.java

```diff
@@ -35,6 +35,22 @@
      */
     String getJdbcUrl(T targetDataSource);
     
+    /**
+     * Get Username.
+     *
+     * @param targetDataSource target data source
+     * @return Username
```

Comment:
`@return Username` should rewrite to `@return username`, please care about case sensitive